.github/workflows/ci.yml

```diff
@@ -84,17 +117,36 @@ jobs:
         with:
           arch: aarch64
           distro: ubuntu22.04
+          env: |
+            VERBOSE: '${{ runner.debug }}'
 
           # Not required, but speeds up builds by storing container images in
           # a GitHub package registry.
           githubToken: ${{ github.token }}
 
           # Set an output parameter `uname` for use in subsequent steps
           run: |
-            apt-get update && apt-get install -y file curl tar gzip
+            apt-get update && apt-get install -y tar gzip curl
             sh app/installer.sh
             file ./kuma-*/bin/kumactl | grep aarch64 &> /dev/null
 
+  installer-sh-mesh:
+    name: Test installer.sh for Kong Mesh
```

Comment:
we usually don't test anything related to KM in `kumahq`. Should this be in https://github.com/Kong/docs.konghq.com ?

Reply:
I think it's fine. If anyone is against it (because it's against some CNCF policy we'll remove it)